### ISSUE TITLE
Cite does not work properly with Zotero 5

### ISSUE THREAD

**Issue by Zefeng-Wu** [beloglazov/zotero-scholar-citations#39]
Extra field is populated with 0000

**Comment by tobidelbruck** [comment]
yes, same for me

**Comment by Thesharing** [comment]
Agree, when I click 'Update citations', the extra fields will be replaced with '00000'

**Comment by lenarddome** [comment]
The same for me. Google blocks it and I am not even able to export bibetex from google scholar as well. The result is the same after the Captcha. The message I got is:

> Your client does not have permission to get URL /scholar.bib?q=info:JmnKBgM_caIJ:scholar.google.com/&output=citation&scisig=AAGBfm0AAAAAW2rfmMAz0IPoLsL3oRLPe37gMKtvL-Qu&scisf=4&ct=citation&cd=-1&hl=en from this server. (Client IP address: xxxxxxxxxxx)
Please see Google's Terms of Service posted at http://www.google.com/terms_of_service.html
That’s all we know.

It is a shame, because the plugin is awesome and I like it very much. Maybe a change of IP will suffice,
or not to do more than 1 at a time.

**Comment by FernaldLab** [comment]
I have the same issue

**Comment by jiangpeiheng** [comment]
This problem I met last year, and it was very annoying. I found that the google treats it as robots. And if I update too many papers in a single time, google scholar will require me to prove I am not a robot when I get to the website.  

I find a simple solution that works for me now. First, you can\`t update too many papers in a shoot. When the number is under 30, it is very safe and very fast. I didn\`t try a larger number. Second, you can look into @tete1030 \`s github website. He fixed this problem in the way that zotero will show `No Citation Data` instead of `00000`. (He also says his version can fix the detection of google robot detection. But I meet a situation that if I try too many in a time, the next time it responds too slow. Then I have to restart the zotero to get it work properly) .  

- This is his [version 1.9.4](https://github.com/tete1030/zotero-scholar-citations), 
- and you can download from here.  https://github.com/tete1030/zotero-scholar-citations/raw/my/builds/zotero-scholar-citations-1.9.4-fx.xpi

**Comment by tete1030** [comment]
This original repo is not properly detecting the robot checking prompt, at least for me, so it will not display any window for the checking procedure. I only fixed the detection and the robot checking window should be popping up now.

Sadly, google will enable additional checking procedure when it detects a large number of queries to their servers within short time. I have tried using random User-agent or clearing cookie for every request, but this just makes it worse.

**Comment by Feiyu-Zhang** [comment]
@tete1030 still not work. My Zotero version is 5.0.64

**Comment by jiangpeiheng** [comment]
@Feiyu-Zhang @tete1030 Just use your version and every time just google a little, it works well for me. At the worst situation, restart it and everything will be fine. It pops the robot checking window and I know when it requires too many. I feel better than the older version.
